packages/vike-react/renderer/+config.ts

```diff
@@ -17,6 +17,9 @@ const toggleSsrRelatedConfig: ConfigEffect = ({ configDefinedAt, configValue })
         env: configValue
           ? { server: true, client: true } // default
           : { client: true }
+      },
+      stream: {
+        env: configValue ? { server: true } : { server: false }
```

Comment:
Can we add a comment here? Like

```
// When the SSR flag is false, stream should be undefined and have no effect
```

Or something like that

Reply:
Sure, I will do that

Reply:
I don't think we need this: checking whether `pageContext.Page ` is defined is enough.

Reply:
What if we change the conditional check inside onRenderHtml to be like this:

```ts
let pageView
if(!pageContext.Page){
pageView = ''
} else {
pageView = !stream ? dangerouslySkipEscape(renderToString(page)) : await renderToStream(page)
}
```

Reply:
LGTM